docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -1,96 +1,9 @@
-[[url-drilldown]]
-=== URL drilldown
+[[url_templating-language]]
+=== URL templating language
```

Comment:
Should we call this just **URL templating** instead of **URL templating language** as this page has information not only about language/syntax, but on variables and various helpers also?

Reply:
This page now also includes a section about disabling url drilldown feature which doesn't fall into `URL templating` :( 
https://kibana_82081.docs-preview.app.elstc.co/guide/en/kibana/master/url_templating-language.html#disable

Reply:
Nice catch! I moved around some content, and now it fits nicely on the first page. Let me know what you think!